plugins/python/plang/Invocation.cpp

```diff
@@ -416,13 +424,22 @@ void Invocation::end(PointView& view, MetadataNode m)
         assert(hasOutputVariable(name));
 
         size_t size = dd->size();
-        void *data = extractResult(name, dd->type());
+        size_t arrSize(0);
+        void *data = extractResult(name, dd->type(), arrSize);
         char *p = (char *)data;
         for (PointId idx = 0; idx < view.size(); ++idx)
         {
             view.setField(d, dd->type(), idx, (void *)p);
             p += size;
         }
+
+        // Set fields for any extra points we've added
+        for (PointId idx = view.size(); idx < arrSize; ++idx)
```

Comment:
this is dumb. We should just iterate to `arrSize` so users can add/remove as necessary